geode-core/src/test/java/org/apache/geode/internal/size/ObjectSizerJUnitTest.java

```diff
@@ -33,13 +34,23 @@ public void test() throws Exception {
     assertEquals(roundup(OBJECT_SIZE), ObjectGraphSizer.size(new TestObject3()));
     assertEquals(roundup(OBJECT_SIZE * 2 + REFERENCE_SIZE),
         ObjectGraphSizer.size(new TestObject3(), true));
-    assertEquals(roundup(OBJECT_SIZE + REFERENCE_SIZE), ObjectGraphSizer.size(new TestObject4()));
-    assertEquals(roundup(OBJECT_SIZE + REFERENCE_SIZE) + roundup(OBJECT_SIZE + 4),
+    if (SystemUtils.isAzulJVM()) {
+      assertEquals(roundup(OBJECT_SIZE + REFERENCE_SIZE + 8), ObjectGraphSizer.size(new TestObject4()));
```

Comment:
This '+ 8' seems suspicious to me - doesn't this indicate that either `OBJECT_SIZE`, `REFERENCE_SIZE` or `ObjectGraphSizer.size` needs to be different on Azul?